apps/emqx/test/emqx_limiter_shared_SUITE.erl

```diff
@@ -172,6 +175,11 @@ t_change_options_from_unlimited(_) ->
     {true, _Client2} = emqx_limiter_client:try_consume(Client1, 1).
 
 t_concurrent(_) ->
+    %% This test fails if log level is debug, so it's important to ensure the correct
+    %% level is set now.
+    #{level := PrevLevel} = logger:get_primary_config(),
+    on_exit(fun() -> emqx_logger:set_primary_log_level(PrevLevel) end),
+    ok = emqx_logger:set_primary_log_level(warning),
```

Comment:
Nit. This doesn't feel like a good practice in general, I suspect there are more test assertions sensitive to increased timings, due to massive volume of logging. Perhaps there should be another assertion in `emqx_cth_suite:verify_clean_suite_state/1` instead?

Reply:
In this case, it was failing not because another test, but because somewhere along running the CT tasks the level was being set to `:debug`.  I suspect it might come from some app inadvertently [starting `Logger`](https://github.com/elixir-lang/elixir/blob/7b20c281d521aa7aa2ad2baa1e9ae6c579d79d0c/lib/logger/lib/logger/app.ex#L35) (maybe during `compile.all`?), but I couldn't spot exactly.

Since this particular test does depend on the assumption that log level is not debug, I made this assumption explicit here.

Reply:
> Ouch! 🫠 Well, to me this sounds like something that definitely should not affect any test code then to be honest, and needs to be fixed on the build system level.

That's why I also set it in `emqx.ct`, to be safe, since it's part of the build system.

> I'm trying to say that there might be few more implicit assumptions this particular test case relies on, as well as other test cases in the code base. Enforcing them on testcase-by-testcase basis might prove ugly in the long term.

Fair.  I think that after [this fix](https://github.com/emqx/emqx/pull/15468/commits/3f2a943fe1e78c6714317b26a86f6a1ce8273da8), this shouldn't be needed.  No harm in leaving it here either, I believe.